pkg/productinfo/alibaba/productinfo_alibaba.go

```diff
@@ -263,3 +310,17 @@ func (e *AlibabaInfoer) GetNetworkPerformanceMapper() (productinfo.NetworkPerfMa
 	nm := newAlibabaNetworkMapper()
 	return nm, nil
 }
+
+func getJson() (DataFromJson, error) {
+	var myClient = &http.Client{Timeout: 10 * time.Second}
+	var dataFromJson DataFromJson
+	r, err := myClient.Get("https://g.alicdn.com/aliyun/ecs-price-info-intl/2.0.1/price/download/instancePrice.json?spm=0.6883001.price.1.741827my27myFB&file=instancePrice.json")
```

Comment:
probably this needs to be made configurable /  maybe coming feom env var through viper